### PR TITLE
Translate built in directives 1

### PR DESCRIPTION
**Pull request by FLarra**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -379,48 +381,54 @@ methods: {
...
+    Ejecute sus pruebas de extremo a extremo, si tiene una. Las <strong>pruebas fallidas</strong> le alertarán sobre cualquier parte de su aplicación que pueda ser afectada por este cambio.
+  </p>
 </div>
 {% endraw %}
 
-### `debounce` Param Attribute for `v-model` <sup>removed</sup>
+### `debounce` Atributo de parámetro para `v-model` <sup>eliminado</sup>
 
-Debouncing is used to limit how often we execute Ajax requests and other expensive operations. Vue's `debounce` attribute parameter for `v-model` made this easy for very simple cases, but it actually debounced __state updates__ rather than the expensive operations themselves. It's a subtle difference, but it comes with limitations as an application grows.
+_Debouncing_ se usa para limitar la frecuencia con la que ejecutamos _requests_ Ajax y otras operaciones costosas. El parámetro de atributo `debounce` de Vue para `v-model` lo hizo fácil para casos muy simples, pero en realidad no dio lugar a __actualizaciones de estado__ en lugar de las costosas operaciones en sí mismas. Es una diferencia sutil, pero viene con limitaciones a medida que aumenta la complejidad de la aplicación.
```
Comment (Tintef) [inline src/v2/guide/migration.md]: What do you about translating `_requests`_ to `solicitudes`?
```suggestion
_Debouncing_ se usa para limitar la frecuencia con la que ejecutamos solicitudes Ajax y otras operaciones costosas. El parámetro de atributo `debounce` de Vue para `v-model` lo hizo fácil para casos muy simples, pero en realidad no dio lugar a __actualizaciones de estado__ en lugar de las costosas operaciones en sí mismas. Es una diferencia sutil, pero viene con limitaciones a medida que aumenta la complejidad de la aplicación.
```

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -461,18 +469,18 @@ new Vue({
 </script>
 {% endraw %}
 
-Using the `debounce` attribute, there'd be no way to detect the "Typing" state, because we lose access to the input's real-time state. By decoupling the debounce function from Vue however, we're able to debounce only the operation we want to limit, removing the limits on features we can develop:
+Usando el atributo `debounce`, no habría forma de detectar el estado "Escribiendo", porque perdemos el acceso al estado en tiempo real de la entrada. Sin embargo, al desacoplar la función _debunce_ de Vue, solo podemos eliminar la operación que queremos limitar, eliminando los límites de las funciones que podemos desarrollar:
```
Comment (Tintef) [inline src/v2/guide/migration.md]: Typo:
```suggestion
Usando el atributo `debounce`, no habría forma de detectar el estado "Escribiendo", porque perdemos el acceso al estado en tiempo real de la entrada. Sin embargo, al desacoplar la función _debounce_ de Vue, solo podemos eliminar la operación que queremos limitar, eliminando los límites de las funciones que podemos desarrollar:
```

**Review by Tintef — COMMENTED** [review]
Just some minor comments, good work 💪

File: src/v2/guide/migration.md (hunk 1 above)
Comment (FLarra) [reply src/v2/guide/migration.md]: I was in doubts regarding that one.. haha

I'm good changing it, but I'd add it to the list of words-mapping so we keep consistent with this kind of translations!

Thank you

**Pull request merged**